src/components/Checkout/InContext/InContextCheckout.vue

```diff
@@ -161,7 +161,14 @@ export default {
 			return appliedCreditsPromoFunds[0] || null;
 		},
 		registerOrLoginHref() {
-			return `/ui-login?force=true&doneUrl=${encodeURIComponent(this.$route.fullPath)}`;
+			const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Add a queryparam, "fromLogin" as part of the return url when using the in-context checkout login